modules/tinymce/src/themes/silver/main/ts/ui/header/InlineHeader.ts

```diff
@@ -246,7 +245,10 @@ export const InlineHeader = (
         Css.set(mainUi.outerContainer.element, 'left', '0px');
         Css.remove(mainUi.outerContainer.element, 'width');
         const w = Width.getOuter(mainUi.outerContainer.element);
-        return Optional.some(w);
+
+        // this check is needed because if the toolbar is rendered outside of the `outerContainer` because the toolbar have `position: "fixed"`
+        // the calculate width isn't correct
+        return w > minimumToolbarWidth ? Optional.some(w) : Optional.none();
```

Comment:
There is an Optionals.someIf that we can use here.

Reply:
changed